### ISSUE TITLE
[Bug]: Incompatibility with Comforts mod

### ISSUE THREAD

**Issue by YuuLC** [terrarium-earth/Handcrafted#38]
### Bug Description

Recipe for sleeping bag in Comforts is the same as the fancy bed in Handcrafted.
Maybe replace one block with cushion?

### How to Reproduce?

1. Have both mods installed (Comforts and Handcrafted)
2. Try to create the fancy bed or sleeping bag by putting three wools in a line
![image](https://github.com/terrarium-earth/Handcrafted/assets/137665060/72bf4c92-2df7-48aa-a8d8-2c9a9095fbf1)

### Expected Behavior

Can create either sleeping bag or fancy bed.

### Version

1.20-2.3.2

### Mod Loader Version

1.20 - 46.0.14

### Mod Loader

Forge

### Code of Conduct

- [X] I have searched the issue tracker and confirmed that the issue has not been reported
- [X] I have checked the FAQ (if one is present) and my issue is not listed
- [X] I have verified that I am using the latest version of the mod
- [X] I have verified that I have installed all the required dependencies for the mod
- [X] I have verified that I do not have Optifine installed. We will close issues if we detect Optifine is in your pack. We cannot fix Optifine issues. Please do not report issues with Optifine present.

**Comment by YuuLC** [comment]
Just noticed, I typed fancy bed, but I meant to type white sheet >.>
Sorry for the confusion!
![image](https://github.com/terrarium-earth/Handcrafted/assets/137665060/c418d7cb-d570-45f8-ae76-df3e7cc575f8)

**Comment by Taleuxss** [comment]
i think their is a mod called polymorph that makes u be able to choose between

**Comment by Ultimushadow** [comment]
You can use Polymorph to enable multiple items to have the same recipe, or you can make a datapack that changes the recipe of one or both items!
*Of course, this is a fix that doesn't involve the mod changing, but at a certain point when you're playing with enough mods, some recipes are bound to overlap. I do recommend always having Polymorph installed just in case, but if you want to make a comprehensive pack without conflicts, that is kind of on you to change the recipes, the mod author can only do so much to handle every other mod in existence. It might be worth making a notes section on the mod page to call to attention mod conflicts though-

**Comment by MarioSMB** [comment]
Why not just allow furniture to be dyed with regular dye items?

**Comment by AlexNijjar** [comment]
should not be up to the modder to fix recipe conflicts

**Comment by MarioSMB** [comment]
Maybe it is not a *responsibiity* of theirs but they can surely consider alternative recipes if theirs conflicts with a popular mod.  

One of those alternatives, for example, could be to simply allow dying the custom beds directly like you do with vanilla beds, saving the need to have a special sheet item.  
In the meantime it is an unfortunate side effect of Minecraft's design that we need to work with recipes that frequently overlap eachother.

**Comment by Ultimushadow** [comment]
I think this is one of those things that people new to modding are just going to have to figure out the hard way. We're all limited by Minecraft, and we all will deal with overlapping recipes at one point or another. It should be standard procedure to suggest Polymorph rather than expect the mod creator to try to juggle recipes or change the way their mod works to accommodate. If you wanted to suggest the idea outside of recipe conflicts I imagine you would not get the immediate pushback.
Bottom line is, alternatives to crafting recipes are not something modders should be considering. Recipe conflicts do not make something incompatible.
